documentation/developer-guide/modules/tooling-guide/pages/samm-cli.adoc

```diff
@@ -26,6 +32,18 @@ alias samm='java -jar /location/to/samm-cli-{esmf-sdk-version}.jar'
 ----
 ====
 
+*For the Windows native binary:*
+. *Download*: download above or visit our github https://github.com/eclipse-esmf/esmf-sdk/releases[releases page] and download executable file for Windows.
```

Comment:
*Download*: Download via above link or visit the repository of the Java SDK which contains the SAMM CLI at github https://github.com/eclipse-esmf/esmf-sdk/releases[releases page] and download the executable file for Windows.